src/NSwag.MSBuild/NSwag.MSBuild.nuspec

```diff
@@ -34,5 +34,6 @@
     <file src="..\NSwag.ConsoleCore\bin\release\netcoreapp2.2\Publish\**" target="tools/NetCore22" />
     <file src="..\NSwag.ConsoleCore\bin\release\netcoreapp3.0\Publish\**" target="tools/NetCore30" />
     <file src="..\NSwag.ConsoleCore\bin\release\netcoreapp3.1\Publish\**" target="tools/NetCore31" />
+    <file src="..\NSwag.ConsoleCore\bin\release\net5.0\Publish\**" target="tools/net50" />
```

Comment:
Should be `tools/Net50` with uppercased N (same as elsewhere). This affects Linux which has case sensitive path handling.

Reply:
You are completely right - fixed that too. Thanks!